R/server.R

```diff
@@ -590,6 +600,156 @@ shinyServer(function(input, output, session) {
         removeModal()
     }, ignoreInit = TRUE)
     
+    
+    #### -------------------------- SLIP DAYS ----------------------------####
+
+    slip_days <- reactiveValues(
+        table = list(),
+        edit = NULL
+    )
+    
+    # Opening category modal to create a NEW SLIP DAYS POLICY
+    observeEvent(input$new_slip_days, {
+        showModal(edit_slip_days_modal)
+        current_edit$slip_days <- NULL
+        updateTextInput(session, "slip_days_policy_name", value = "Slip Days Policy 1")
+        updateNumericInput(session, "num_slip_days", value = 0)
+        updateSelectInput(session, "slip_days_order", selected = "chronological")
+        if (!is.null(assign$table)) {
+            updateSelectizeInput(session, "assignments", choices = assign$table$assignment, selected = NULL)
+        }
+    })
+    
+    #Edit a slip day policy
+    observeEvent(input$edit_slip_days, {
+        req(current_edit$slip_days)
+        policy <- slip_days$table[[current_edit$slip_days]]
+        showModal(edit_slip_days_modal)
+        updateTextInput(session, "slip_days_policy_name", value = policy)
+        updateNumericInput(session, "num_slip_days", value = policy$num_slip_days)
+        updateSelectInput(session, "slip_days_order", selected = policy$order)
+        updateSelectizeInput(session, "assignments", choices = assign$table$assignment, selected = policy$assignments)
```

Comment:
Same comment here too: let's try assignments+categories in the dropdown

Reply:
Maybe, in the same vein as `get_assignments(gs)` and `get_id_cols(gs)`, we should also have `get_categories(policy)` and `get_assignments(category)` in the package